js/lib/appUrlUtil.js

```diff
@@ -24,6 +24,13 @@ module.exports.fileUrl = (filePath) => {
   return encodeURI('file://' + fileUrlPath)
 }
 
+module.exports.chromeUrl = (filePath = '') => {
```

Comment:
would be good to have some unit tests for this

Reply:
added in 1bc13ba

Reply:
can we use a relative path instead and make it filesystem agnostic?